docs/housing_search.md

```diff
@@ -33,13 +33,20 @@ ElasticSearch will be updated by a Lambda function, holding business logic, that
 * A result should act as a navigation tool to deeper more meaningful information.
 * Search will most likely be the first thing a user does in order to find information.
 * Search should be available for any data set (property, transaction, document).
+* The search results should be concise enough to be digestable in a single workflow and not excessive
```

Comment:
Please add(to be more clear): If the exact match is not required then false will be passed and get the current result.

Reply:
Added this wording